manage-github.py

```diff
@@ -74,9 +74,17 @@ def protect_branch(repo, branch=None, **kwargs):
             current_protection = convert_protection(protected_branch.get_protection())
         except GithubException as e:
             if e.status == 404:
+                # new repo no protection set
                 protection = kwargs
             else:
-                raise
+                yield client.Change(
```

Comment:
might be worth adding a comment here, equivalent to line 77, saying e.g. `# private repo forked into our org` ?

Reply:
good call

Reply:
added